test/tests/depletion/depletion.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+import os, shlex, sys
+from MooseControl import MooseControl
+
+# All of these imports are awful and we'll figure
+# out a better way to do it before merging
+try:
+   import openmc
+except:
+    openmc_contrib = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..', 'contrib', 'openmc'))
+    if os.path.exists(openmc_contrib):
+        sys.path.append(openmc_contrib)
+    import openmc
+
+
+import openmc.lib
+import openmc.deplete
+from openmc.deplete import CoupledOperator
+
+try:
+    from cardinal_operator import CardinalOperator
+except:
+    cardinal_contrib = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..', 'python'))
```

Comment:
see above, this could also be solved by requiring the `PYTHONPATH` to be set to `CARDINAL_HOME/python` in the installation notes.

Reply:
A warning similar to the one about `NEKRS_HOME` could be provided perhaps.